### PR TITLE
docs(README): Document how to test an in-development version of Baseplate in Snoodev

### PR DESCRIPTION
**Pull request by KTAtkinson**
Closes # 

## 📜 Details
[Design Doc]()

[Jira]()

## ✅ Checks

- [ ] CI tests (if present) are passing
- [ ] Adheres to code style for repo
- [ ] Contributor License Agreement (CLA) completed if not a Reddit employee

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,25 @@ actions on push.
 Specific hooks can be [temporarily disabled](https://pre-commit.com/#temporarily-disabling-hooks).
 
 You can install hooks only for a specific step (i.e. [pre-push](https://pre-commit.com/#pre-commit-during-push)).
+
+# Testing in Snoodev
```
Comment (nsheaps) [inline README.md]: 🔕 Not against it here, and these instructions seem fine, but seems a little strange to me for this to be here but testing sidecars to be [in the wiki](https://reddit.atlassian.net/wiki/spaces/BAS/pages/2410939140/Baseplate+Sidecars#How-do-I-test-the-sidecars%3F)

File: README.md [hunk 2]
```diff
@@ -47,3 +47,25 @@ actions on push.
...
+--- a/requirements.txt
++++ b/requirements.txt
+-baseplate==2.5.7
++git+https://github.com/reddit/baseplate.py@<ref>#egg=baseplate
+```
+
+In this case the ref can be either a commit hash or a branch name. After making
+this update you can check if the service works in Snoodev.
+
+> **Warning**
```
Comment (nsheaps) [inline README.md:69]: TIL markdown syntax formats this

File: README.md [hunk 3]
```diff
@@ -47,3 +47,25 @@ actions on push.
...
+> Never deploy this change to production. Production should always use a Baseplate
+> version that has been released and published to pypi.
```
Comment (nsheaps) [inline README.md]:
```suggestion
> semantic version that has been tagged, and released.
```

**Review by nsheaps — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (KTAtkinson) [reply README.md]: Note that talks about how to release sidecars and then test them, not how to test them locally. Also currently this approach doesn't work with sidecars.

File: README.md (hunk 2 above)
Comment (KTAtkinson) [reply README.md:69]: I had to look it up :D

**Review by ghirsch-reddit — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RyanBalfanz) [reply README.md]: 🔕 nit: Multiple top-level headings in the same document

Comment (RyanBalfanz) [inline README.md]: Multiple top-level headings in the same document

```suggestion
## Testing in Snoodev
```

Comment (RyanBalfanz) [reply README.md]: There are other headings and lint errors in this file, but they are not related to this change so this is the only suggestion I'm offering at this time.

Comment (RyanBalfanz) [reply README.md]: https://github.com/reddit/baseplate.py/pull/757

**Pull request merged**